ci/jenkins/test.sh

```diff
@@ -519,6 +524,19 @@ function deliver_antrea_windows {
     echo "==== Finish building and delivering Windows images ===="
 }
 
+function copy_kind_test_image {
+    common_images=("registry.k8s.io/e2e-test-images/agnhost:2.40" \
```

Comment:
here also similar issue can occur, when the images are changed in the e2e code, so either you should prefer loading the image using the variables defined in e2e code, so that when they are updated the updates are reflected here as well.

Reply:
ok, will try this.